.github/workflows/publish.yml

```diff
@@ -23,16 +23,16 @@ jobs:
         id: release
         uses: manovotny/github-releases-for-automated-package-publishing-action@v2.0.1
 
-      # Perform installs, run tests, run a build step, etc. here, as needed.
-      - run: yarn
-
```

Comment:
Is this a merge conflict change?

Reply:
Not sure, but it's not present in the PR any more. I've handled all the merge conflicts, and now the actual intended changes are all that remain.